### ISSUE TITLE
Keyboard navigation does not work in menus when using an item component

### ISSUE THREAD

**Issue by nienow** [angular/components#17660]
#### Reproduction

https://stackblitz.com/edit/angular-lxzdru

Steps to reproduce:
1. Define a custom component to use as the menu items
2. Click on the menu button OR tab to the button and hit enter
3. When the menu appears, the first menu item is not focused. And hitting the down arrow does not focus the next item.
 
#### Expected Behavior

After opening the menu, you can use down/up arrows to focus a menu item

#### Actual Behavior

Using the up/down arrows do not focus menu items

#### Environment

- Angular: 8.2.9
- CDK/Material: 8.2.3
- Browser(s): Chrome
- Operating System:  Windows

I determined that this functionality broke starting in Material version 8.1.0. Before that version, it worked correctly.

**Comment by nienow** [comment]
Duplicate of 18553
